### PR TITLE
V8: Fix for MediaPickerValueConverter when config is only images

### PR DESCRIPTION
**Pull request by bergmania**
Fixes https://github.com/umbraco/Umbraco-CMS/issues/4827

### CODE REVIEW

File: src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs [hunk 1]
```diff
@@ -53,26 +65,31 @@ private bool IsOnlyImagesDataType(PublishedDataType dataType)
...
 
-        public override object ConvertSourceToIntermediate(IPublishedElement owner, PublishedPropertyType propertyType, object source, bool preview)
+        public override object ConvertSourceToIntermediate(IPublishedElement owner, PublishedPropertyType propertyType,
+            object source, bool preview)
         {
             if (source == null) return null;
 
             var nodeIds = source.ToString()
-                .Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries)
+                .Split(new[] {","}, StringSplitOptions.RemoveEmptyEntries)
```
Comment (zpqrtbnk) [inline src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs]: well, we've all used a space around curlies until now... so I know if I work on this file, I'll revert this without even thinking - and I think even R# would do it for me... so, are you sure about it?

File: src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs [hunk 2]
```diff
@@ -31,15 +39,19 @@ public override Type GetPropertyValueType(PublishedPropertyType propertyType)
...
+                    ? typeof(IEnumerable<>).MakeGenericType(ModelType.For(ImageTypeAlias))
+                    : typeof(IEnumerable<IPublishedContent>)
+                : isOnlyImages
+                    ? ModelType.For(ImageTypeAlias)
+                    : typeof(IPublishedContent);
         }
 
         public override PropertyCacheLevel GetPropertyCacheLevel(PublishedPropertyType propertyType)
-            => PropertyCacheLevel.Snapshot;
+        {
```
Comment (zpqrtbnk) [inline src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs]: why?

Comment (bergmania) [reply src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs]: Just, did a file format in Rider. Maybe I had to tweak the settings. Right now it uses default settings that should be based on what is used most in the code base. Can tweak this.

File: src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs (hunk 1 above)
Comment (bergmania) [reply src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs]: Just, did a file format in Rider. Maybe I had to tweak the settings. Right now it uses default settings that should be based on what is used most in the code base. Can tweak this.

**Pull request merged**